packages/usa-button/src/usa-button.stories.js

```diff
@@ -62,3 +64,17 @@ Secondary.args = SecondaryContent;
 
 export const Unstyled = Template.bind({});
 Unstyled.args = UnstyledContent;
+
+export const TestLinksAsButtons = TestTemplateLinks.bind({});
+TestLinksAsButtons.args = {
+  modifier: "",
+  text: "Link styled as button",
+  is_demo: true,
```

Comment:
Small tweak, but it should be safe to remove this line since we declare it as the defaultValue in [line 26](https://github.com/uswds/uswds/pull/5435/files#diff-33b264a45949b5dbda9916d0c19947e50ba5bfe9feb66f9aa46562488872e73eL25)
```suggestion
```

Reply:
Ooh thanks for pointing that out! Removed in [fb698eb](https://github.com/uswds/uswds/pull/5435/commits/fb698eb16744f410b4615af245f56604ae5caf24)